<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionImage</class>
 <widget class="QMainWindow" name="gestionImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestionImage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../logo_lycee.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../Pictures/xl_9666-Le-BTS-Systemes-Numeriques-SN-forme-les-Technicien.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>158</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>montre_img_1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShow">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>158</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>montre_img_2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hide">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>158</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>cache</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>charge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>décharge</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="cin_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>nom fichier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>entrée</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>160</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>copie de fichier </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RSAchiff">
       <property name="text">
        <string>chiffrement RSA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SHA">
       <property name="text">
        <string>SHA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AES">
       <property name="text">
        <string>chiffrement AES</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="cin_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>données</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="RSAdechiff">
       <property name="text">
        <string>dechiff RSA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AESdechiff">
       <property name="text">
        <string>dechiff AES</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
